components/engine_rocks/src/event_listener.rs

```diff
@@ -100,6 +100,45 @@ impl rocksdb::EventListener for RocksEventListener {
         STORE_ENGINE_INGESTION_PICKED_LEVEL_VEC
             .with_label_values(&[&self.db_name, info.cf_name()])
             .observe(info.picked_level() as f64);
+        if info.picked_level() < 6 {
+            let file_path = info.internal_file_path().to_str().unwrap();
+            let reader = RocksSstReader::open_with_env(file_path, None);
+            match reader {
+                Ok(reader) => {
+                    let iter_opt = IterOptions::new(None, None, false);
+                    let iter = reader.iter(iter_opt);
+                    if let Err(e) = iter {
+                        warn!(
+                            "SST can not ingest to L6";
+                            "level" => info.picked_level(),
+                            "file" => file_path,
+                            "iter_error" => e.to_string(),
+                        );
+                        return;
+                    }
+                    let mut iter = iter.unwrap();
+                    _ = iter.seek_to_first();
+                    let min_key = log_wrappers::hex_encode_upper(iter.key());
+                    _ = iter.seek_to_last();
+                    let max_key = log_wrappers::hex_encode_upper(iter.key());
+                    warn!(
+                        "SST can not ingest to L6";
```

Comment:
What's the purpose of printing min/max information? 
I think it does not worth the run-time cost here ----that is needs to read the file, seek first and last. It could make ingest SST even slower.

Reply:
Sorry this PR is only made to debug a problem. I'll add DNM in the title